### ISSUE TITLE
Adding to reading list via image does not have correct text.

### ISSUE THREAD

**Issue by whitecat** [alexpdraper/reading-list#76]
If an image is selected when adding a link what text should we add? 
Right now I believe the text is being pulled from the page not the image. This needs to be debugged and identify what is actually being added to the title.

**Comment by alexpdraper** [comment]
We could check for alt text on the image? Or if the `<a>` has a title. Otherwise maybe just the hostname from the URL?

**Comment by whitecat** [comment]
I agree with using the Alt Text, second using title, and last using just the host name.
